### PR TITLE
Add warning with instructions to use new preview location (github.io no longer updated)

### PR DESCRIPTION
**Pull request by Zlatkovsky**
Added a warning to the bottom of each of the files.

Note that looks like GitHub doesn't render the minified files well (it thinks it's a large diff), whereas in reality it's just a newline being added at the bottom of the file, followed by something like:

```
console.warn(
    'You are referencing "<XYZ>.js" from a location that is being discontinued.' +
    '\n' +
    'Please update the HTML page for your task pane to use the following URL instead:' +
    '\n' +
    'https://appsforoffice.microsoft.com/lib/preview/hosted/<XYZ>.js'
);
```

@keyur32 or @akrantz , please merge in once you've update the Yo Office templates to also make use of the preview endpoint.

### CODE REVIEW

File: lib/custom-functions-runtime.debug.js [hunk 1]
```diff
@@ -5626,4 +5626,12 @@ OSF._OfficeAppFactory = (function OSF__OfficeAppFactory() {
...
         }
     }("undefined" != typeof self ? self : this);
-} ]);
\ No newline at end of file
+} ]);
+
+console.warn(
+    'You are referencing "custom-functions-runtime.debug.js" from a location that is being discontinued.' +
+    '\n' +
+    'Please update the HTML page for your custom function runner to use the following URL instead:' +
```
Comment (akrantz) [inline lib/custom-functions-runtime.debug.js]: suggest: "custom functions" instead of "custom function runner"

File: lib/office.debug.js [hunk 2]
```diff
@@ -1713,3 +1713,11 @@ OSF._OfficeAppFactory = (function OSF__OfficeAppFactory() {
         getWindowName: function OSF__OfficeAppFactory$getWindowName() { return _windowName; }
     };
 })();
+
+console.warn(
+    'You are referencing "office.debug.js" from a location that is being discontinued.' +
+    '\n' +
+    'Please update the HTML page for your task pane to use the following URL instead:' +
```
Comment (akrantz) [inline lib/office.debug.js]: suggest: "Office Add-in" instead of "task pane"

**Comment by Zlatkovsky** [comment]
I think it's best to have someone from the Documentation team (or @keyur32) to review the strings, to make sure that they are consistent with what the docs describe these concepts as.  After that, I can change it.

For the part about "Office Add-ins" vs "task pane" -- I called out the taskpane intentionally, since a CF is also part of an Office Add-in.  The idea was that if someone is referencing Office.js in their taskpane in order to call AsyncStorage API, they should stop doing that, and switch to the new preview location.

@keyur32, waiting on you to review the strings and chime in.

**Comment by akrantz** [comment]
Yes, but office.js can also be used for the ribbon appcommands in addition to the task pane. It could also apply if custom functions used office.js but they only use custom-functions-runtime.js or office-runtime.js.

**Comment by zlatko-michailov** [comment]
Is this PR still required, or can we close it?

**Comment by Zlatkovsky** [comment]
@zlatko-michailov , I'm guessing at this point this is old enough that we can just close it without merging.  (If someone has been referencing the github.io site this whole time, they probably would have noticed other issues...).  Closing.

**Pull request closed without merging**